src/actions/actions_makeshift.py

```diff
@@ -0,0 +1,55 @@
+from talon import Module, actions
+from dataclasses import dataclass
+
+
+@dataclass
+class MakeshiftAction:
+    term: str
+    identifier: str
+    vscode_command_id: str
+    pre_command_sleep: float = 0
+    post_command_sleep: float = 0
+
+
+# NOTE: Please do not change these dicts.  Use the CSVs for customization.
+# See https://github.com/pokey/cursorless-talon/blob/main/docs/customization.md
+makeshift_actions = [
+    MakeshiftAction("define", "revealDefinition", "editor.action.revealDefinition"),
+    MakeshiftAction(
+        "type deaf", "revealTypeDefinition", "editor.action.goToTypeDefinition"
+    ),
+    MakeshiftAction("hover", "showHover", "editor.action.showHover"),
+    MakeshiftAction("inspect", "showDebugHover", "editor.debug.action.showDebugHover"),
+    MakeshiftAction(
+        "quick fix", "showQuickFix", "editor.action.quickFix", pre_command_sleep=0.3
+    ),
+    MakeshiftAction("reference", "showReferences", "references-view.find"),
+    MakeshiftAction("rename", "rename", "editor.action.rename", post_command_sleep=0.1),
+]
```

Comment:
Once we support extra args in csv, we could prob migrate these to the new custom action setup, right?  We'd need to add the following columns:

- `pre_command_sleep`
- `post_command_sleep`
- `restore_selection`

Reply:
I actually fought about that. The different is that makeshift actions don't actually jump back. Indifference to things like custom actions and paste to.